src/main/java/frc/robot/RobotContainer.java

```diff
@@ -82,11 +89,14 @@ public RobotContainer(RobotRunType runtimeType) {
                 s_Vision = new Vision(state, VisionSimPhoton.partial(driveSimulation));
```

Comment:
You need to put something here even if its just a placeholder using `elevator = new Elevator(new ElevatorIO() {});`. Or else if we run simulation and try to interact with elevator, the code will crash null pointer.